File: bench/cases/gcd1m/main.py
def gcd(a, b):
    while b != 0:
        t = b
        b = a % b
        a = t
    return a


total = 0
target = 1234567
for i in range(1, 1000001):
    total += gcd(i, target)

print(total)

File: bench/cases/gcd1m/main.rs
fn gcd(mut a: u64, mut b: u64) -> u64 {
    while b != 0 {
        let t = b;
        b = a % b;
        a = t;
    }
    a
}

fn main() {
    let mut total: u64 = 0;
    let target: u64 = 1234567;
    for i in 1..=1000000_u64 {
        total += gcd(i, target);
    }
    println!("{}", total);
}
